sdk/tests/core/test_az_http.c

```diff
@@ -560,22 +560,87 @@ static void test_http_response(void** state)
     }
   }
 
-  az_span response_span = AZ_SPAN_FROM_STR( //
-      "HTTP/1.1 200 Ok\r\n"
-      "Content-Type: text/html; charset=UTF-8\r\n"
-      "\r\n"
-      // body
-      EXAMPLE_BODY);
-
-  az_http_response response = { 0 };
-  az_result const result = az_http_response_init(&response, response_span);
-  assert_true(result == AZ_OK);
-
-  // add parsing here
+  // get body directly
```

Comment:
Why do we repeat this comment here and below on line 576? Update this one to match the other "test case description" comments.
```suggestion
  // Processing valid response
```